packages/rolldown/src/options/input-options.ts

```diff
@@ -136,6 +136,7 @@ export interface InputOptions {
   jsx?: JsxOptions
   watch?: WatchOptions | false
   dropLabels?: string[]
+  keepNames?: boolean
```

Comment:
Should add this to schema too.

Reply:
https://github.com/rolldown/rolldown/blob/4e356ed4c6010a1b1a1eae65e1c7db3653e3565a/crates/rolldown_testing/_config.schema.json?plain=1#L323-L328 it is added in previous pr.